docs/manual/manual.adoc

```diff
@@ -580,30 +580,63 @@ command-line argument. You can use `--results-arf` to generate a Result DataStre
 $ oscap xccdf eval --benchmark-id benchmark_id --results-arf arf-results.xml scap-ds.xml
 --------------------------------------------------------------------------------------
 
-==== Result STIG Viewer
+==== Results for STIG Viewer
 
-If you want to import the XCCDF scan results to DISA STIG Viewer but your Rule IDs don't
-match the DISA's ones, you can use the `--stig-viewer` command-line argument along with
-a special reference in your Rules to generate XCCDF result files that can be imported by
-DISA STIG Viewer.
+DISA STIG Viewer is a graphical user interface (GUI) application that enables
+easy viewing of SCAP formatted Security Technical Implementation Guides
+(STIGs).  For more information on DISA STIG Viewer go to
+https://public.cyber.mil/stigs/srg-stig-tools/[SRG / STIG Tools].
 
---------------------------------------------------------------------------------------
-$ oscap xccdf eval --profile stig-rhel7-disa --stig-viewer results-stig.xml ssg-rhel7-ds.xml
---------------------------------------------------------------------------------------
+OpenSCAP can generate results compatible with DISA STIG Viewer when evaluating
+SCAP content not provided by DISA that uses different Rule IDs than official
+DISA STIG, for example `scap-security-guide` or third party content.
 
-Each rule in the input XCCDF must contain a reference to its STIG Rule ID, and its 
-href attribute must be exactly `http://iase.disa.mil/stigs/Pages/stig-viewing-guidance.aspx`.
+To be able to produce results compatible with DISA STIG Viewer, each rule in
+the input SCAP source data stream must contain a reference to a STIG Rule ID,
+and its `href` attribute must be either
+`http://iase.disa.mil/stigs/Pages/stig-viewing-guidance.aspx` or
+`https://public.cyber.mil/stigs/srg-stig-tools/`.
 
 For example:
 ----
 <Rule id="rpm_verify_permissions">
   ...
-  <reference href="http://iase.disa.mil/stigs/Pages/stig-viewing-guidance.aspx">SV-86473r2_rule</reference>
+  <reference href="https://public.cyber.mil/stigs/srg-stig-tools/">SV-86473r2_rule</reference>
   ...
 </Rule>
 ----
 
-For more information on DISA STIG Viewer click link:http://iase.disa.mil/stigs/Pages/stig-viewing-guidance.aspx[here].
+A good example of SCAP source data stream that meets this assumption is the
+`/usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml` provided by
+`scap-security-guide` RPM package. We will use this file to demonstrate its
+usage with DISA STIG Viewer.
+
+1) Scan your system using `oscap` with `--stig-viewer` option.
+
+----
+$ oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_stig --stig-viewer results-stig.xml /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml
+----
+
+2) Get the STIG, eg. from https://public.cyber.mil/stigs/downloads/[STIGs
```

Comment:
Download a STIG file of your choice, for example, from the link:https://public.cyber.mil/stigs/downloads/[STIGs Document Library], and extract it. The version of the STIG must conform to the version of the `xccdf_org.ssgproject.content_profile_stig` profile.